### ISSUE TITLE
Twitch new Transcode Options

### ISSUE THREAD

**Issue by Xeala** [streamlink/streamlink#59]
Since TwitchCon some streams will get new transcode options, so livestreamer couldn't pick up the best one on these, if i had "default-stream best" in my config file.

http://puu.sh/rtnvj/2526c5a084.png

**Comment by bastimeyer** [comment]
This was fixed by chrippa/livestreamer#1429... The PR has been merge here:
https://github.com/streamlink/streamlink/blame/master/src/streamlink/plugin/plugin.py#L40

```bash
$ streamlink -p mpv twitch.tv/twitch best
[cli][info] Found matching plugin twitch for URL twitch.tv/twitch
[cli][info] Available streams: audio, 144p30 (worst), 240p30, 360p30, 480p30, 540p30, 720p30, 720p60, 1080p60 (best)
[cli][info] Opening stream: 1080p60 (hls)
[cli][info] Starting player: mpv
[cli][info] Player closed
[cli][info] Stream ended
```

**Comment by gravyboat** [comment]
Since this was already fixed I'm going to close this.
